src/render-html/css.js

```diff
@@ -253,9 +253,8 @@ hr {
 .message_inline_image img,
 .twitter-image img {
   width: 100%;
-  max-height: 40vh;
-  object-fit: cover;
-  border-radius: 4px;
+  height: 30vh;
```

Comment:
Can we add a comment here explaining that it's important to have a fixed height?  I feel like this is a land mine that someone could step on in the future.

Also, how carefully did you test this?  It's not clear to me whether we need to remove the `width` setting, and/or whether the correct attribute to set a fixed height on is the `img` tag or the wrapper div `.message_inline_image`.  I think the actual behavior we want is to set the height on `message_inline_image` div

Reply:
Yeah, I did test it.
If we set both `height` and `width` it is a potential problem with forcing that aspect ratio.
That is why `object-fit` makes sure we don't do that.

I think we can improve the html produced by the server to be more 'semantic' and thus more flexible to styling, which will likely improve the reliability of that style.

Reply:
Yeah, the plan is to change the server HTML when we have thumbnailing in place.